noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -597,10 +1098,14 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST4">
         <title>Series period containment</title>
+        <description>
+            Tests whether the Series' created and finalized dates
```

Comment:
`Tests whether the created and finalized dates of all series are within the archival period specified in arkivuttrekk.xml`